src/components/Hero.jsx

```diff
@@ -38,7 +39,7 @@ const Hero = () => {
         </div>
 
         <h1 className="font-poppins font-semibold ss:text-[68px] text-[52px] text-white ss:leading-[80px] leading-[80px] w-full">
-          <span className="text-gradient">Parth Mittal </span>
+          <span className="text-gradient">{aboutMe[0].name} {aboutMe[0].lastName}</span>
```

Comment:
Change according to the object notation.

Reply:
done!